src/Symfony/Component/Console/Color.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console;
+
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class Color
```

Comment:
I love this feature! However, this `Color` class looks like it should be `Style` or `TextStyle` ... because it defines several colors and some text style properties (bold, underscore, ...)

I guess we want to keep things simple, but for your consideration, we could split this `Color` class in `Style` or `TextStyle` + `Color` classes.

Reply:
This is a Color in the context of Console. And that means foreground + background + options. I don't see the benefit of adding more classes (that won't be useful by themselves).